source/managing-teams/what-engineering-managers-do.html.md

```diff
@@ -0,0 +1,94 @@
+---
+title: What Engineering Managers do
+meta_description: >
+  Engineering managers are the point of contact for clients at Nebulab, as well as looking after
+  the growth and well-being of their team.
+---
+
+At Nebulab, we strongly believe in the value of competent, supportive management.
+
+When the company was 10 people, the Directors could look after each project personally, interacting
```

Comment:
This is the general direction we want to move towards. Don't just explain the state of things, but explain the reason behind them, so that people can understand the principles that we've followed when making certain decisions.

Reply:
I think it's fine providing some extra context

Reply:
I think I agree with @matteogalliani here mainly for two reasons:

- I feel the playbook should state the current status of the company and not its history (otherwise it would be called historybook)
- I think we weren't even directors back then as we had a completely different set of responsibilities and this feels to me like we are over-simplifying everything (we were probably more EMs more than directors)

All of the above said... It's a short paragraph, so not the end of the world if it stays in.

Reply:
I've given this a bit more thought, I think you guys might be right. Explaining the context behind a decision doesn't necessarily mean telling people about the entire history of the company. In this case, having EMs is an industry best practice, and I don't think that there's much to justify here. I'll remove the paragraph.